docs/build_node.md

```diff
@@ -305,22 +319,22 @@ byte[] best_block_hash = // <insert current chain tip block hash>;
 ChannelManagerConstructor channel_manager_constructor = new ChannelManagerConstructor(
   LDKNetwork.LDKNetwork_Bitcoin, UserConfig.default(), best_block_hash,
   block_height, keys_manager.as_KeysInterface(), fee_estimator, chain_monitor,
-  router, tx_broadcaster, logger);
+  tx_broadcaster, logger);
```

Comment:
Huh?

Reply:
https://github.com/lightningdevkit/ldk-garbagecollected/blob/main/src/main/java/org/ldk/batteries/ChannelManagerConstructor.java#L46 I don't see the router argument there anymore?

Reply:
Yea, it'll be added soon :)

Reply:
Ah! Is it going to be optional? Otherwise, what should people put there if they skipped step 4?

Reply:
Its `@Nullable`, yes.